src/css/embed.scss

```diff
@@ -114,6 +114,11 @@ $brexit-control-button-size: 28px;
     min-height: $baseline * 2;
     padding-bottom: $baseline * 1.5;
 }
+.brexit__content {
```

Comment:
have you checked that this doesn't mess up any other atom types? `.brexit__content` is used inside `.brexit--nav-controls`: https://github.com/guardian/interactive-brexit-companion/blob/4c221d02731fb44afd791e9ad40284c30f0469f6/src/css/embed.scss#L202